### ISSUE TITLE
Pax multiprocessing should not start if worker fails to initialize

### ISSUE THREAD

**Issue by JelleAalbers** [XENON1T/pax#405]
While building run 1958, the following error crashed the output plugin of the eventbuilder:

```
Traceback (most recent call last):
  File "/home/xedaq/anaconda3/envs/pax_jelle/lib/python3.4/multiprocessing/process.py", line 254, in _bootstrap
    self.run()
  File "/home/xedaq/anaconda3/envs/pax_jelle/lib/python3.4/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/home/xedaq/jelle/pax/pax/core.py", line 286, in __init__
    self.action_plugins = [self.instantiate_plugin(x) for x in action_plugin_names]
  File "/home/xedaq/jelle/pax/pax/core.py", line 286, in <listcomp>
    self.action_plugins = [self.instantiate_plugin(x) for x in action_plugin_names]
  File "/home/xedaq/jelle/pax/pax/core.py", line 373, in instantiate_plugin
    instance = getattr(plugin_module, name_class)(this_plugin_config, processor=self)
  File "/home/xedaq/jelle/pax/pax/plugin.py", line 31, in __init__
    y = self.startup()
  File "/home/xedaq/jelle/pax/pax/FolderIO.py", line 230, in startup
    os.mkdir(self.output_dir)
FileExistsError: [Errno 17] File exists: '/data/xenon/raw/160805_1909'
```

Perhaps more importantly, the trigger continued reading and building events after this, and just halted when the output queue was filled. Fortunately event clearing happens as an output plugin (has to, since events must appear in order), else we would have lost some LED data.

**Comment by JelleAalbers** [comment]
Same with 1960

**Comment by tunnell** [comment]
#2038

**Comment by JelleAalbers** [comment]
Sorry, I meant this issue for the fact that any crash upon initialization of the worker plugins is not properly communicated to the master.

**Comment by JelleAalbers** [comment]
This is addressed in #439, also for local multiprocessing.
